### PR TITLE
include boost chrono

### PR DESCRIPTION
**Pull request by fwyzard**
* build boost chrono
* add a boost_chrono tool to scram

### CODE REVIEW

**Comment by fwyzard** [comment]
I would like to use boost chrono for its `boost::chrono::thread_clock`

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
Do you think we can merge this in by pre5 ?
It's necessary to have per-thread cpu time measurements, which in turn is needed to support multithreading in the FastTimerService .

**Comment by smuzaffar** [comment]
@fwyzard , I do not see any issue with it.
@Dr15Jones , @davidlange6 any objections?

**Comment by Dr15Jones** [comment]
I'm fine with it.

File: boost-toolfile.spec [hunk 1]
```diff
@@ -28,6 +28,15 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/boost.xml
 </tool>
 EOF_TOOLFILE
 
+# boost_chrono toolfile
+cat << \EOF_TOOLFILE >%i/etc/scram.d/boost_chrono.xml
+<tool name="boost_chrono" version="@TOOL_VERSION@">
+  <info url="http://www.boost.org"/>
+  <lib name="@BOOST_CHRONO_LIB@"/>
+  <use name="boost"/>
```
Comment (smuzaffar) [inline boost-toolfile.spec]: @fwyzard , looks like boost_chrono  needs boost_system
``` ldd libboost_chrono.so | grep boost
libboost_system.so.1.57.0 => not found
```
so please add dependency on ```<use name="boost_system"/>``` boost_system any way depends on boost.

can you also please update the version number to 1.2 of this spec?

**Comment by fwyzard** [comment]
done !

**Comment by fwyzard** [comment]
please test

**Pull request merged**